images/build/Dockerfiles/ltsVersions.Dockerfile

```diff
@@ -77,9 +55,9 @@ ENV DOTNET_RUNNING_IN_CONTAINER=true \
     DOTNET_USE_POLLING_FILE_WATCHER=true \
 	NUGET_XMLDOC_MODE=skip \
     DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1 \
-	NUGET_PACKAGES=/var/nuget
+	NUGET_PACKAGES=/opt/nuget
 
-RUN mkdir /var/nuget
+RUN mkdir /opt/nuget
```

Comment:
you can attach this mkdir command to a previous or post run statements as well

Reply:
I am not worried about this since it is an intermediate stage